openreview/journal/webfield/editorsInChiefWebfield.js

```diff
@@ -303,6 +315,17 @@ var formatData = function(
     var cameraReadyTask = null;
     var cameraReadyVerificationTask = null;
 
+    if (aeRecommendationInvitation) {
+      var recommendationCount = aeRecommendations[submission.id] || 0;
+      tasks.push({
+        id: aeRecommendationInvitation.id,
+        cdate: aeRecommendationInvitation.cdate,
+        duedate: aeRecommendationInvitation.duedate,
+        complete: recommendationCount >= 3,
+        replies: Array(recommendationCount).fill(1, 0, recommendationCount)
```

Comment:
If you are filling the whole array, it's more clear to just say `fill(1)` and omit the other args.